content/en/docs/instrumentation/python/automatic/example.md

```diff
@@ -273,31 +276,30 @@ If those headers are available, they will be included in your span:
 
 ```json
 {
-    "attributes": {
-        "http.request.header.user-agent": [
-            "Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0)"
-        ],
-        "http.request.header.accept_encoding": [
-            "gzip, deflate, br"
-        ],
-        "http.response.header.last_modified": [
-            "2022-04-20 17:07:13.075765"
-        ],
-        "http.response.header.content_type": [
-            "text/html; charset=utf-8"
-        ]
-    }
+  "attributes": {
+    "http.request.header.user-agent": [
+      "Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0)"
+    ],
+    "http.request.header.accept_encoding": ["gzip, deflate, br"],
+    "http.response.header.last_modified": ["2022-04-20 17:07:13.075765"],
+    "http.response.header.content_type": ["text/html; charset=utf-8"]
+  }
```

Comment:
Note the reformatting of the JSON code block. /cc @cartermp et al.

Reply:
I am OK with that.